### PR TITLE
feat: fincube-9 .env files edited and fixed

### PR DESCRIPTION
**Pull request by mashiat0808**
- .env files converted to .env.local
- example.env converted to .env.example

### CODE REVIEW

File: backend/.gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 */node_modules
-*/.env
\ No newline at end of file
+*/.env.*
```
Comment (FahimDev) [inline backend/.gitignore]: Don't Do that. Cz your env.example will also get ignored.

File: backend/dao-service/.gitignore [hunk 2]
```diff
@@ -3,6 +3,8 @@
 /node_modules
 /build
 
+
+.env.*
```
Comment (FahimDev) [inline backend/dao-service/.gitignore]: Don't Do that. Cz your env.example will also get ignored.

**Comment by mashiat0808** [comment]
- .gitignore files edited

**Review by FahimDev — APPROVED** [review]
LGTM

**Pull request merged**